tests/ocsp.ml

```diff
@@ -103,8 +106,11 @@ let test_simple_responder () =
     let responder_id = `ByName responder_dn in
     Fmt.epr "keytype = %a" Key_type.pp (Private_key.key_type responder_key);
```

Comment:
This is the only case where there is output to stderr in this test module. Is that intentional or was this intended to be elided like line 70 above?

Reply:
oops, I guess this is leftover debug code, and should be removed.

Reply:
done in cd23240